content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-github-classroom.md

```diff
@@ -0,0 +1,41 @@
+---
+title: About GitHub Classroom
+intro: 'Teachers and school administrators can use {% data variables.product.prodname_classroom %} to create virtual classrooms, make and edit assignments, automatically grade assignment submissions, and more.'
+type: overview
+topics: 
+  - Fundamentals
+versions:
+  fpt: '*'
+---
+
+## About {% data variables.product.prodname_classroom %}
+
+{% data variables.product.prodname_classroom %} is a teaching tool that lets teachers and school administrators create and manage digital classrooms and assignments. You can create assignments for individual students or groups, set due dates, and track assignments on your teacher dashboard. Additionally, {% data variables.product.prodname_classroom %} has many features that simplify tasks like providing feedback, grading assignments, and integrating your existing teaching tools.
```

Comment:
Minor suggestion as it reads oddly otherwise 🤔 
```suggestion
{% data variables.product.prodname_classroom %} is a teaching tool that lets teachers and school administrators create and manage digital classrooms and assignments. You can create assignments for individual students or groups of students, set due dates, and track assignments on your teacher dashboard. Additionally, {% data variables.product.prodname_classroom %} has many features that simplify tasks like providing feedback, grading assignments, and integrating your existing teaching tools.
```